Documentation/spelling_wordlist.txt

```diff
@@ -80,6 +80,7 @@ Meltwater
 Menlo
 MiB
 Monnet
+Multicast
```

Comment:
Lowercase entries in the list are case-insensitive, so `Multicast` shouldn't be needed.